### PR TITLE
Renaming, take 2

### PR DESCRIPTION
**Pull request by cjyabraham**
Renaming done again now that we've updated docs to aragonOS 4.  The following names were changed:
* Aragon Package Manager ==> aragonPM
* Aragon CLI ==> aragonCLI
* Kit ==> Template
* Aragon Core ==> Aragon client
* aragon.js ==> aragonAPI
* Aragon UI ==> aragonUI
* Aragon ID ==> aragonID

### CODE REVIEW

File: docs/cli-usage.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ aragon init [app-name] [boilerplate]
 - `boilerplate`: the Github repo name or alias for a boilerplate to set up your app. The currently available boilerplates are:
 
 	- `react`: this boilerplate contains a very basic Counter app and a webapp for interacting with it. It showcases the end-to-end interaction with an Aragon app, from the contracts to the webapp.
-	- `react-kit`: it is a variation of the `react` boilerplate that also comes with a DAO Kit which will allow for using your app to interact with other Aragon apps like the Voting app. You can read more about DAO Kits [here](kits-intro.md).
+	- `react-kit`: it is a variation of the `react` boilerplate that also comes with a DAO Template which will allow for using your app to interact with other Aragon apps like the Voting app. You can read more about DAO Templates [here](kits-intro.md).
```
Comment (sohkai) [inline docs/cli-usage.md]: I think we shouldn't change the "Kit" convention just yet, as it'll need a push across many different repos (and a new CLI release) to align them to "template". We also need to decide on their exact naming ("DAO Template", "Aragon Template", "Organization Template", etc.).

cc @luisivan

File: docs/glossary.md [hunk 2]
```diff
@@ -7,16 +7,16 @@ sidebar_label: Glossary
...
-- **APM Repository**: Smart contract deployed inside APM that keeps track of the versions for a package.
-- [**aragon.js**](/docs/aragonjs-intro.html): Javascript library to interact with aragonOS by handling transaction pathing, upgradeability and state of the contracts.
-- [**Aragon UI**](/docs/aragonui-intro.html): Aragon-native toolkit of UI components for decentralized apps.
-- [**Aragon client**](http://app.aragon.org): Client to interact with aragonOS-powered decentralized organizations. It implements a signer with transaction pathing, notifications and a sandboxed environment for aragonOS-based apps using aragon.js.
+- [**aragonPM**](/docs/apm.html): Decentralized package manager based on aragonOS that handles upgreadability of smart contracts and arbitrary data blobs, such as webapps.
+- **aragonPM Repository**: Smart contract deployed inside aragonPM that keeps track of the versions for a package.
+- [**aragonAPI**](/docs/aragonjs-intro.html): Javascript library to interact with aragonOS by handling transaction pathing, upgradeability and state of the contracts.
```
Comment (sohkai) [inline docs/glossary.md]: We need to reword this bit, as `aragonAPI` is an umbrella name for all user-facing API packages, and `@aragon/api` will be one particular implementation in javascript. This is also where the specs (#23) would live under.

How about:


- aragonAPI: Standard set of APIs and specifications used to interact with aragonOS-powered contracts by handling transaction pathing, upgradeability, and contract state. Reference implementations in specific languages include:
  - [JavaScript](/docs/aragonjs-intro.html)

The rest of the changes should make it clear that it's the JavaScript version of aragonAPI.

**Review by sohkai — APPROVED** [review]
Looking good!

I don't think we're ready to make the change to use "templates" yet, so I'd stash those changes and apply them after :).

File: docs/cli-usage.md (hunk 1 above)
Comment (cjyabraham) [reply docs/cli-usage.md]: reverted 8b05e3c8ebf9d2b732e7b9476bc42f5eb5d04f39

**Review by luisivan — APPROVED** [review]
Great work @cjyabraham!!

**Review by Smokyish — APPROVED** [review]

**Pull request merged**